src/endpoints/transactions.rs

```diff
@@ -15,9 +15,6 @@ pub(crate) use get::Request as Get;
 #[non_exhaustive]
 #[derive(Deserialize, Debug)]
 pub struct Transaction {
-    /// The account balance after the transation
-    pub account_balance: i64,
```

Comment:
wow i didn't see this. so it's *never* published anymore? is that documented anywhere? in the community forums maybe?

Reply:
The best I have managed to find is this [issue](https://github.com/monzo/docs/issues/127) and this [community thread](https://community.monzo.com/t/account-balance-at-time-of-transaction/111153).